python/instrumentation/openinference-instrumentation-mistralai/src/openinference/instrumentation/mistralai/_chat_wrapper.py

```diff
@@ -144,15 +146,7 @@ def _parse_args(
         request_data: Dict[str, Any] = {}
```

Comment:
after binding arguments, use `apply_defaults` to ensure that we capture defaulted arguments as well

Reply:
^ bound defaults, changed tests to check only explicitly set parameters